pkgs/top-level/all-packages.nix

```diff
@@ -36670,8 +36670,7 @@ with pkgs;
 
   windowlab = callPackage ../applications/window-managers/windowlab { };
 
-  windowmaker = callPackage ../applications/window-managers/windowmaker { };
-  dockapps = callPackage ../applications/window-managers/windowmaker/dockapps { };
+  dockapps = callPackage ../by-name/wi/windowmaker/dockapps { };
```

Comment:
Oh, this is something I didn't expect :sweat_smile: 

I guess it's not a problem, it will fall under a future refactoring of package sets

Reply:
Windowmaker dockapps live in a limbo.
In theory they can run in any window manager that provides dockapp support.
However, there are only Windowmaker and Afterstep that provides this.